src/common.rs

```diff
@@ -190,9 +190,15 @@ impl KeylimeConfig {
                 Err(e) => warn!("Could not load TPM data"),
             }
         }
-        let run_revocation = bool::from_str(
-            &config_get("cloud_agent", "listen_notfications")?.to_lowercase(),
-        )?;
+        // There was a typo in Python Keylime and this accounts for having a version
+        // of Keylime installed that still has this typo. TODO: Remove later
+        let listen = match &config_get("cloud_agent", "listen_notifications")
```

Comment:
Perhaps this could be a little simpler with `or_else`: `config.get(..., "listen_notifications").or_else(|| config.get(..., "listen_notfications"))?.to_lowerase()`?

Reply:
Thanks @ueno , I like that syntax better.